concept/validation/schema-validation.feature

```diff
@@ -503,243 +747,332 @@ Feature: Schema validation
     Given entity(ent2) set supertype: ent1
     Given transaction commits
 
-    When session opens transaction of type: write
+    When connection open schema transaction for database: typedb
     When entity(ent2) set plays: rel0:role0
-    When entity(ent1) set plays: rel1:role1 as rel0:role0; throws exception
+    When entity(ent2) get plays(rel0:role0) set annotation: @card(1..1)
+    When entity(ent2) get plays(rel0:role0) set override: rel0:role0
+    When entity(ent1) set plays: rel1:role1
+    When transaction commits
+
+    When connection open schema transaction for database: typedb
+    When entity(ent1) get plays(rel1:role1) set override: rel0:role0
+    Then transaction commits; fails
 
 
   Scenario: A thing-type may not be moved in a way that its plays declarations are hidden by an override
-    Given typeql define
```

Comment:
Rewritten it to concept api as this step was a `typeql define` because "it was faster to write"